src/lib.rs

```diff
@@ -585,14 +585,21 @@ pub fn list() -> Vec<(String, String)> {
 }
 
 #[doc(hidden)]
-pub fn eval<R, F: FnOnce(Option<String>) -> R>(name: &str, f: F) -> Option<R> {
+pub fn eval<R, F: FnOnce(Option<String>) -> R>(
+    name: &str,
+    f: F,
+    cond: impl FnOnce() -> bool,
```

Comment:
Use generic instead to not introduce unnecessary rustc version requirement.

Reply:
`impl Trait` was stabilized in 1.26. We are currently using Edition 2018  which forces the minimum Rust version to be 1.31. "Version requirement" is not a good reason to reject this line.

Reply:
Got it.